crates/ere-zisk/src/lib.rs

```diff
@@ -642,7 +616,7 @@ mod tests {
     #[test]
     fn test_prove() {
         let program = basic_program();
-        let zkvm = EreZisk::new(program, ProverResourceType::Cpu);
+        let zkvm = EreZisk::new(program, ProverResourceType::Gpu);
```

Comment:
How does this work in CI?

Reply:
Ah I forget to clean it up when testing GPU.

It works in CI because currently runner can't run the `prove` due to insufficient disk space, so this is not caught 🥲